source/transactions/tests/unified/backpressure-retryable-commit.yml

```diff
@@ -0,0 +1,222 @@
+      useMultipleMongoses: false
+      observeEvents:
+        - commandStartedEvent
+  -
+    database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database_name transaction-tests
+  -
+    collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection_name test
+  -
+    session:
+      id: &session0 session0
+      client: *client0
+
+initialData:
+  -
+    collectionName: *collection_name
+    databaseName: *database_name
+    documents: []
+tests:
+  - description: commitTransaction retries if backpressure labels are added
+    runOnRequirements:
+      - serverless: forbid
+    operations:
+      - object: testRunner
+        name: failPoint
+        arguments:
+          client: *client0
+          failPoint:
+            configureFailPoint: failCommand
+            mode:
+              times: 2
+            data:
+              failCommands:
+                - commitTransaction
+              errorLabels:
+                - RetryableError
+                - SystemOverloadedError
+              errorCode: 112
+      - object: *session0
+        name: startTransaction
+      - object: *collection0
+        name: insertOne
+        arguments:
+          session: *session0
+          document:
+            _id: 1
+        expectResult:
+          $$unsetOrMatches:
+            insertedId:
+              $$unsetOrMatches: 1
+      - object: *session0
+        name: commitTransaction
+    expectEvents:
+      - client: *client0
+        events:
+          - commandStartedEvent:
+              command:
+                insert: test
+                documents:
+                  - _id: 1
+                ordered: true
+                readConcern:
+                  $$exists: false
+                lsid:
+                  $$sessionLsid: *session0
+                txnNumber:
+                  $numberLong: "1"
+                startTransaction: true
+                autocommit: false
+                writeConcern:
+                  $$exists: false
+              commandName: insert
+              databaseName: *database_name
+          - commandStartedEvent:
+              command:
+                commitTransaction: 1
+                lsid:
+                  $$sessionLsid: *session0
+                txnNumber:
+                  $numberLong: "1"
+                startTransaction:
+                  $$exists: false
+                autocommit: false
+                writeConcern:
+                  $$exists: false
+              commandName: commitTransaction
+              databaseName: admin
+          - commandStartedEvent:
+              command:
+                commitTransaction: 1
+                lsid:
+                  $$sessionLsid: *session0
+                txnNumber:
+                  $numberLong: "1"
+                startTransaction:
+                  $$exists: false
+                autocommit: false
+                writeConcern:
```

Comment:
None of the transaction retries should have writeConcerns applied, right?  There is no write concern set on the client or the session.